test/control-plane/services/dual-stack/state1.yaml

```diff
@@ -0,0 +1,198 @@
+apiVersion: v1
```

Comment:
what does `state1` filename means? Could we perhaps give a more descriptive filename?

Reply:
This used to be events1.yaml, etc. but since we also need to do deletes I ended up making these represent the whole state (not including objects created in init.yaml), hence the "stateN.yaml" naming. I was thinking it'd be cleaner to do deletes explicitly,  but didn't yet have time to do that. I'll look into this and the non-golden test example today.